src/scss/vendors/_vendors.scss

```diff
@@ -31,6 +31,45 @@
 	}
 }
 
+// uio constrast focus and hover
+@each $theme, $foregroundColor, $backgroundColor in $uio-themes {
```

Comment:
This loop and the loop at line 102 can be combined into one. It's unnecessary to iterate the same element list twice.